integration/tests/cook/test_basic.py

```diff
@@ -1106,6 +1116,10 @@ def test_error_while_creating_job(self):
         self.assertEqual(resp.status_code, 500)
 
     @pytest.mark.xfail
+    @unittest.skipIf(util.has_ephemeral_hosts(), 'If the cluster under test has ephemeral hosts, then it is generally '
+                                                 'a bad idea to use HOSTNAME EQUALS constraints, because it can cause '
+                                                 'the process responsible for launching hosts to launch hosts that '
+                                                 'never get used')
```

Comment:
Can we have this string in a constant somewhere and refer to the constant in the annotation?

Reply:
Done